src/components/WwwFrame/LendMenu/TheLendMenu.vue

```diff
@@ -55,6 +63,22 @@ export default {
 		LendListMenu,
 		LendMegaMenu,
 	},
+	created() {
+		if (!this.$isServer) {
+			const { enabled } = getExperimentSettingCached(this.apollo, lendMenuExpKey);
+			if (enabled) {
+				trackExperimentVersion(this.apollo, this.$kvTrackEvent, 'TopNav', lendMenuExpKey);
+				getExperimentSettingAsync(this.apollo, lendMenuExpKey)
+					.then(({ version }) => {
+						this.weighedCategoriesExp = version === 'shown';
```

Comment:
Because `b` was used as the variant key in the SettingsManager entry, `b` should be checked for here instead of `shown`

Reply:
You're right, sorry ! I solved this.